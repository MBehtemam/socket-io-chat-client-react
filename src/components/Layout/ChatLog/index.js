import styled from "styled-components";

const ChatLog = styled.section`
  height: ${props => props.theme.chatContainerHeight};
  background-color: ${props => props.theme.chatContainerBackground};
  display: flex;
  flex-direction: column;
`;
export default ChatLog;
