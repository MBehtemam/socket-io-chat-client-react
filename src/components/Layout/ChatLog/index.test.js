import React from "react";
import styled from "styled-components";
import renderer from "react-test-renderer";
import "jest-styled-components";
import ChatLog from "./index";

describe("Test suits for ChatLog", () => {
  test("should works", () => {
    const tree = renderer.create(<ChatLog />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
