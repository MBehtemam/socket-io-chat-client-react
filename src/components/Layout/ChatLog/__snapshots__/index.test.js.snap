// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test suits for ChatLog should works 1`] = `
.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
}

<section
  className="c0"
/>
`;
